ex_feels/lib/ex_feels_web/templates/layout/app.html.eex

```diff
@@ -21,4 +21,453 @@
 
     <script src="<%= static_path(@conn, "/js/app.js") %>"></script>
   </body>
+
+  <script src="https://d3js.org/d3.v3.min.js"></script>
```

Comment:
lets install d3 via npm instead of making a network request

Reply:
good point